ui/src/components/timetables/vehicle-schedule-details/VehicleRouteTimetableSection.tsx

```diff
@@ -24,20 +24,29 @@ const testIds = {
 };
 
 export const VehicleRouteTimetableSection = ({
-  route,
+  routeId,
   initiallyOpen = false,
 }: Props): JSX.Element => {
   const { t } = useTranslation();
   const [isOpen, toggleIsOpen] = useToggle(initiallyOpen);
   const { activeView, setShowPassingTimesByStop } = useTimetablesViewState();
 
+  const routeResult = useGetRouteDetailsByIdQuery({
```

Comment:
I'm pretty sure you don't need all the fields of the GetRouteDetailsById query here, please restrict it a bit!